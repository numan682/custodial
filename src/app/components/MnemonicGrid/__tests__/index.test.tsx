import * as React from 'react'
import { render } from '@testing-library/react'
import { MnemonicGrid } from '..'

describe('<FromMnemonic/>', () => {
  it('should match snapshot', () => {
    const page = render(
      <MnemonicGrid
        mnemonic={[
          'planet',
          'beelieve', // Should mark typo
          'session',
        ]}
      />,
    )
    expect(page.container.firstChild).toMatchSnapshot()
  })
})
