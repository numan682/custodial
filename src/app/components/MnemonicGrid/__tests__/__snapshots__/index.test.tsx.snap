// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FromMnemonic/> should match snapshot 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  margin: 6px;
  background-color: #33333310;
  color: #444444;
  border-bottom: solid 1px rgba(0,0,0,0.33);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 6px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-right: 12px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  max-width: 100%;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 {
  display: grid;
  box-sizing: border-box;
  grid-template-columns: repeat( auto-fill,minmax(14ch,1fr) );
}

.c3 {
  font-size: 18px;
  line-height: 24px;
}

@media only screen and (max-width:768px) {
  .c1 {
    margin: 3px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    border-bottom: solid 1px rgba(0,0,0,0.33);
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    padding: 3px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding-right: 6px;
  }
}

<span
  class="notranslate"
  translate="no"
>
  <div
    class="c0"
    data-testid="mnemonic-grid"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
          style="user-select: none;"
        >
          1
          .
        </span>
      </div>
      <div
        class="c4"
      >
        <span
          class="notranslate"
          translate="no"
        >
          <strong
            style="white-space: pre;"
          >
            planet
          </strong>
        </span>
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
          style="user-select: none;"
        >
          2
          .
        </span>
      </div>
      <div
        class="c4"
      >
        <span
          class="notranslate"
          translate="no"
        >
          <strong
            style="white-space: pre; text-decoration: red wavy underline;"
          >
            beelieve
          </strong>
        </span>
      </div>
    </div>
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
          style="user-select: none;"
        >
          3
          .
        </span>
      </div>
      <div
        class="c4"
      >
        <span
          class="notranslate"
          translate="no"
        >
          <strong
            style="white-space: pre;"
          >
            session
          </strong>
        </span>
      </div>
    </div>
  </div>
</span>
`;
